types/telegram-web-app/index.d.ts

```diff
@@ -32,6 +32,10 @@ interface WebApp {
      * The color scheme currently used in the Telegram app. Either “light” or
      * “dark”. Also available as the CSS variable var(--tg-color-scheme).
      */
+    platform: string;
```

Comment:
JSDoc should go above the property it describes; this edit has made the JSDoc for color move out of place.

Reply:
I will fix it

Reply:
Fixed @MKRhere